tools/wptrunner/wptrunner/wptrunner.py

```diff
@@ -48,8 +48,8 @@ def setup_logging(*args, **kwargs):
 def get_loader(test_paths, product, debug=None, run_info_extras=None, chunker_kwargs=None, **kwargs):
     if run_info_extras is None:
         run_info_extras = {}
-
-    run_info = wpttest.get_run_info(kwargs["run_info"], product,
+    python_version = sys.version_info.major
```

Comment:
Why are we reading this here and passing it all the way down into the `RunInfo` class vs just reading it from `sys.version_info` in the `RunInfo` class itself?

Reply:
I had made it unnecessarily complicated. Code updated. Thank you!